src/blocks.tsx

```diff
@@ -0,0 +1,350 @@
+import * as React from 'react';
+import {sprinkles} from './sprinkles.css';
+import Stack from './stack';
+import * as styles from './blocks.css';
+import {Text2, Text3, Text5, Text8} from './text';
+import {vars} from './skins/skin-contract.css';
+import Inline from './inline';
+import Box from './box';
+import ProgressBar from './progress-bar';
+import classNames from 'classnames';
+
+import type StackingGroup from './stacking-group';
+import type Image from './image';
+import type Tag from './tag';
+import type {RendersNullableElement} from './utils/renders-element';
+
+interface BlockContentProps {
+    title?: string;
+    description?: string | Array<string>;
+}
+
+const BlockContent: React.FC<BlockContentProps> = ({title, description}) => {
+    const normalizedDescription =
+        description && Array.isArray(description) ? (
+            description.map((paragraph, i) => (
+                <Text2 regular color={vars.colors.textSecondary} key={i}>
+                    {paragraph}
+                </Text2>
+            ))
+        ) : (
+            <Text2 regular color={vars.colors.textSecondary}>
+                {description}
+            </Text2>
+        );
+
+    return (
+        <div className={styles.column}>
+            {title && (
+                <Text3 regular color={vars.colors.textPrimary}>
+                    {title}
+                </Text3>
+            )}
+
+            {!!description && normalizedDescription}
+        </div>
+    );
+};
+
+interface RowBlockProps {
+    title?: string;
+    stackingGroup?: RendersNullableElement<typeof StackingGroup>;
+    description?: string;
+    'aria-label'?: string;
+}
+
+export const RowBlock: React.FC<RowBlockProps> = ({
+    title,
+    stackingGroup,
+    description,
+    'aria-label': ariaLabel,
+}) => {
+    return (
+        <div
+            style={{
+                display: 'flex',
+                alignItems: 'center',
+            }}
+            aria-label={ariaLabel}
+        >
+            {(title || stackingGroup || description) && (
```

Comment:
I find the behavior of "not having the title prop, the stackingGroup is not showing" quite odd.
https://tinyurl.com/2lalc6qz

I would appreciate the flexibility to use this component with just the title + progress, or solely with the stackingGroup + progress, for example

I plan to discuss this matter with the Vivo design team to propose a change.